formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/FormstackSarService.scala

```diff
@@ -77,4 +86,58 @@ object FormstackSarService extends FormstackSar with LazyLogging {
     else
       decode[FormstackFormSubmissionsResponse](response.body)
   }
+
+  private def submissionsById(
```

Comment:
this method is called submissionsById but returns FormstackSubmissionResponse. Naming is hard, i think we should try and give it a better name.

Reply:
agree - done